blscosi/protocol/sub_protocol.go

```diff
@@ -445,3 +447,16 @@ func (p *SubBlsCosi) checkIntegrity() error {
 
 	return nil
 }
+
+// sendToChildren does not collect error messages,
+// but doesn't block on nodes with a non-rejecting firewall in front of them.
+func (p *SubBlsCosi) sendToChildren(msg interface{}) {
+	for _, c := range p.Children() {
+		go func(tn *onet.TreeNode) {
+			err := p.SendTo(tn, msg)
+			if err != nil {
+				log.Warnf("Error while sending to children: %v", err)
+			}
+		}(c)
+	}
+}
```

Comment:
You know I really don't like dropping errors (I even feel that this `func` comment was just for me :smile: )

What about smth like? That gives you a stream of error, closing it when all childs are done, so you can both know the end of the broadcast and if any error happened :)
```suggestion
func (p *SubBlsCosi) sendToChildren(msg interface{}) <-chan error {
	ret := make(chan error)

	childsRets := make(chan error)
	go func() {
		for range p.Children() {
			if err := <-childsRets; err != nil {
				ret <- xerrors.Errorf("send to children: %v", err)
			}
		}
		close(ret)
	}()

	for _, c := range p.Children() {
		go func(tn *onet.TreeNode) {
			childsRets <- p.SendTo(tn, msg)
		}(c)
	}

	return ret
}
```